src/Schema/DataObject/Plugin/QueryFilter/Filters/InFilter.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+
+namespace SilverStripe\GraphQL\Schema\DataObject\Plugin\QueryFilter\Filters;
+
+use SilverStripe\GraphQL\Schema\DataObject\Plugin\QueryFilter\ListFieldFilterInterface;
+use SilverStripe\ORM\Filterable;
+
+/**
+ * A query filter that filters records by the presence of a value in an array
+ */
+class InFilter implements ListFieldFilterInterface
+{
+    /**
+     * @inheritDoc
+     */
+    public function apply(Filterable $list, string $fieldName, $value): iterable
+    {
+        return $list->filter($fieldName . ':ExactMatch', (array) $value);
```

Comment:
```suggestion
        return $list->filter($fieldName, (array) $value);
```

Don't think we need the `ExactFilter` clause here. Maybe we should throw an exception if we get something that's not an array.